charts/sonarqube/templates/change-admin-password-hook.yml

```diff
@@ -43,6 +43,10 @@ spec:
       containers:
       - name: {{ template "sonarqube.fullname" . }}-change-default-admin-password
         image: {{ default "curlimages/curl:latest" .Values.curlContainerImage }}
+        {{- if $securityContext := .Values.initContainers.securityContext }}
```

Comment:
i missed that initially but here again. reusing the `initContainers` scope is misleading

Reply:
Created a account.securityContext value